.remarkrc.js

```diff
@@ -25,13 +25,24 @@ exports.plugins = [
   [require('remark-lint-prohibited-strings'), [2, [
     { no: 'master' },
     { no: 'slave' },
+    { no: 'blacklist' },
+    { no: 'whitelist' },
+    { no: 'graylist' },
+    { no: 'dark(-|\s)?greylist' },
+    { no: 'light(-|\s)?greylist' },
+    { no: 'tribal(-|\s)?knowledge' },
+    { no: 'tribal(-|\s)?wisdom' },
     { no: 'tribal' },
     { no: 'tribalism' },
+    { no: 'tribal(-|\s)?division'},
     { no: 'nitty(-|\s)?gritty' },
     { no: 'voodoo' },
     { no: 'off(-|\s)?the(-|\s)?reservation' },
+    { no: 'the(-|\s)?natives'},
```

Comment:
Can line 42 be updated to include both with and without the the? Something like `(the(-|\s)?)?natives`?